source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -1,215 +1,170 @@
 .. _wp-install:
 
-Create a Clear Linux WordPress\* server
+Create a Clear Linux\* WordPress\* server
 #######################################
 
-Prerequisites
-=============
+This tutorial shows you how to install the WordPress components on your |CL| LAMP server. At the end of :ref:`Creating a Clear Linux based web server <web-server-install>`, you created a
```

Comment:
Sorry. It was my error. We need to also add an asterisk to first instance of Linux in the body text too.  
Please add asterisk as follows: `:ref: `Creating a Clear Linux\* based web server.`

Reply:
Let me clarify some rather important things:
The asterisk should only be added on the first instance in the body of the document of a specific trademarked named owned by an entity other than Intel.
Asterisks in headings should be avoided as much as possible! They create problems with cross-references. @mvincerx @deb-intel You are now seeing one example of said problems.
Additionally, the cross-reference you are adding has a label, `web-server-install`. It is incorrect to add the heading's text here since the \:ref\:\`web-server-install\` role will automatically use it for the link upon rendering.
In short, replace the `ref` role with \:ref\:\`web-server-install\` and everything should render properly.

Reply:
IMHO, I believe our team would be better served if the use of asterisks were documented in our Contributor Guidelines. This would be more fair to external and internal contributors.  

As for the :ref: directive, there are good reasons to use "link text" instead of just using a link :ref:`<web-server-install>` to the html page. For example, it would also be correct to write: 

[...] At the end of :ref: `the web server installation guide <web-server-install>`, you created a [...]
This style is widely used in CL documentation. 

Note: Formatting for the above may not render correctly here. 

On the other hand, I see your point on avoiding redundancy in the header when using :ref:.